roles/as_code/lib/engineer.rb

```diff
@@ -1,16 +1,45 @@
 require_relative './shared_expectations/core_engineer_expectations'
 
 JobSpec::Role.definition 'Engineer' do
-  description 'Our software delivery teams are the core of our business. They are usually comprised of a Lead Engineer, a Delivery Manager, and one or more Engineers. As an Engineer we expect someone:'
+  description <<~DESCRIPTION
+    Our engineers build software that make our customers happy, prefer problem solving over task completing, love learning from colleagues, and constantly strive to be nice humans.
+
+    ## What does the job entail?
+
+    We primarily write and deliver custom software to our customers. We build solutions in a variety of languages and platforms, historically Ruby on Rails, we tend to use React, JavaScript, Elixir, and Clojure these days. We will usually be responsible for setting up a customer’s infrastructure either on AWS with tools like Ansible and Terraform. Sometimes we opt for a Platform as a Service like Heroku. We will make sure there is a continuous delivery pipeline running our automated test suites before deploying to a staging environment, and a manual button for going to production.
```

Comment:
either on AWS OR...?

Reply:
Continuous Delivery

Reply:
describing the pipeline down to manual push to production seems oddly prescriptive in this context?